.micropico

```diff
@@ -0,0 +1,3 @@
+{
+  "info": "This file is just used to identify a project folder."
```

Comment:
do we need to include this?

Reply:
I think it is necessary for micropico to work correctly.

Reply:
ok, maybe we can move it to the `group-3` directory?